CRM/Utils/Token.php

```diff
@@ -1714,13 +1707,29 @@ public static function replaceMultipleContributionTokens(string $separator, stri
    * @return string token replacement
    */
   public static function getMembershipTokenReplacement($entity, $token, $membership) {
-    self::_buildMembershipTokens();
+    $supportedTokens = [
```

Comment:
Confirmed that this matches the effective array here https://github.com/civicrm/civicrm-core/blob/master/CRM/Core/SelectValues.php#L526

Reply:
thanks